src/base/bittorrent/torrentimpl.cpp

```diff
@@ -96,6 +100,102 @@ namespace
             entryList.emplace_back(setValue.toStdString());
         return entryList;
     }
+
+    lt::announce_entry makeNativeAnnouncerEntry(const QString &url, const int tier)
+    {
+        lt::announce_entry entry {url.toStdString()};
+        entry.tier = tier;
+        return entry;
+    }
+
+#if (LIBTORRENT_VERSION_NUM >= 20000)
+    TrackerEntry fromNativeAnnouncerEntry(const lt::announce_entry &nativeEntry, const lt::info_hash_t &hashes)
```

Comment:
Doesn't seem like something public. Only TorrentImpl knows how to correctly fill TrackerEntry info. Otherwise, it will give the false impression that it can be created from any combination of `lt::announce_entry` and `lt::info_hash_t`.